src/uikit.cs

```diff
@@ -314,6 +324,92 @@ partial interface NSFileProviderExtension {
 	
 	    [Export ("stopProvidingItemAtURL:")]
 	    void StopProvidingItemAtUrl (NSUrl url);
+
+#if XAMCORE_2_0 && IOS
+
+		[iOS (11,0)]
+		[Export ("itemForIdentifier:error:")]
+		[return: NullAllowed]
+		XamCore.FileProvider.INSFileProviderItem GetItem (/*[BindAs (typeof (NSFileProviderItemIdentifier))]*/ NSString identifier, out NSError error);
+
+		// Inlining NSFileProviderExtension (NSFileProviderActions) so we get asyncs
+
+		[iOS (11,0)]
+		[Async]
+		[Export ("importDocumentAtURL:toParentItemIdentifier:completionHandler:")]
+		void ImportDocument (NSUrl fileUrl, string parentItemIdentifier, Action<XamCore.FileProvider.INSFileProviderItem, NSError> completionHandler);
```

Comment:
And it is, it is just to workaround a generator bug that I actually need to file it in bugzilla, generator has issues generating the trampolines if I do not do that way

```diff
+#if XAMCORE_2_0 && IOS
 +using XamCore.FileProvider;
 +#endif
```